topics/digital-humanities/tutorials/open-refine-tutorial/tutorial.md

```diff
@@ -0,0 +1,473 @@
+
+**Why this dataset?** It is credible, openly published, and realistically messy—ideal for practising problems scholars encounter at scale. Records include a **Categories** field populated from the **Powerhouse Museum Object Names Thesaurus (PONT)**, a controlled vocabulary reflecting Australian usage. The tutorial deliberately surfaces common quality issues—blank values that are actually stray whitespace, duplicate rows, and multi-valued cells separated by the pipe character `|` (including edge cases where **double pipes** `||` inflate row counts)—so we can practice systematic inspection before any analysis. During cleaning, you will compute sanity checks (after de-duplication, the dataset drops to **XXXX** unique records; a facet reveals **XXXX** distinct categories and **XXXX** items with no category). Without careful atomization and clustering, these irregularities would bias statistics, visualizations, and downstream reconciliation.
+
+We suggest that you download the data from the Zenodo record as explained below. This helps us with the reproducibility of the results.
+
+> <hands-on-title>Upload your data</hands-on-title>
+>
+> 1. Create a new history for this tutorial and name it "Powerhouse Museum — OpenRefine"
+> 2. Import the file from [Zenodo]({{page.zenodo_link}}):
+>
+>    ```
+>    https://zenodo.org/records/17047254
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the dataset: "**Powerhouse Museum metadata**."
+> 4. Ensure that the datatype is "tsv". Otherwise, use convert datatype.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+{: .hands_on}
+
+# Use OpenRefine to explore and clean your dataset
+
+The users will familiarize themselves with the museum's metadata. In the next step, they will clean the metadata using various steps to enhance its quality and accessibility.
+
+## Start OpenRefine
+
+> <hands-on-title>Opening the dataset with OpenRefine</hands-on-title>
+>
+> 1. Open the {% tool [OpenRefine](interactive_tool_openrefine) %}: Working with messy data  
+>    - *"Input file in tabular format"*:  `openrefine-phm-collection.tsv`  
+>
+> 2. Click on "Run Tool".  
+>
+>    <p align="center">
+>      <img src="openrefine.png" alt="OpenRefine tool interface in Galaxy">
+>    </p>
```

Comment:
I can see why you added screenshots here, but now that there is a good intro to Galaxy tutorial in the DH topic, I would suggest removing the Galaxy screenshots, since the interface changes a lot and people might be confused if it doesnt look the same. And we can assume they are familiar with the Galaxy basics at this point.